src/views/waypoint/home/components/heading/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface HeadingProps {
+  __heading_slug: string
+  heading: string
+  level: number
```

Comment:
This might allow us to avoid the `as React.ElementType` in the component itself:
```suggestion
  level: 1 | 2 | 3 | 4 | 5 | 6
```